setup.py

```diff
@@ -41,6 +41,7 @@
         'ocrd>=3.0.0a1',
         'click',
         'scipy',
+        'python-Levenshtein>=0.25.1',
```

Comment:
note: that's already on my fork's branch (without version req) – so this will conflict.

Reply:
If you mean the `bertsky:fix-alpha-shape` fork, I do not see it there.

Reply:
I'm sorry – what I actually meant was rapidfuzz instead of python-levensthein:

https://github.com/bertsky/ocrd_cis/commit/5282092997ad6b2b53a3cef8c3c96fbb27066682

But it will be easy to resolve the conflict.